blues/simulation.py

```diff
@@ -282,6 +291,42 @@ def setStateConditions(self):
         self.setSimState('md', 'state0', md_state0)
         self.setSimState('nc', 'state0', nc_state0)
 
+    def _getSimulationInfo(self):
+        partNC = 0.2 / (1.0/ self.opt['nstepsNC'])
```

Comment:
Maybe quick comments explaining what these are? `partNC` is particularly opaque to me. The others I can mostly guess.

Reply:
I'm going to leave this in for now without adding docstrings. I'm working on implementing a reporter for the NCMC simulation to eventually replace this.